<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>makeJob</class>
 <widget class="QDialog" name="makeJob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>massage</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;me&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>606</x>
     <y>23</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>me</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>23</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>My net</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>706</x>
     <y>23</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>jobs</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>23</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Make-Job</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>job title:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <underline>true</underline>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>job location:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <underline>true</underline>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>job type:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
       <underline>true</underline>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>company:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
       <underline>true</underline>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="title">
     <string>Workplace Type:</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
       <underline>true</underline>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
